test/test_io.py

```diff
@@ -2301,6 +2303,146 @@ def test_nondefault_nbins_many_5(self):
                                 scale='linear'),
             [bins1, np.arange(262145) - 0.5, bins2])
 
+class TestFCSAttributesChannelLabels(unittest.TestCase):
+    """
+    Test correct extraction, functioning, and slicing of channel label.
+
+    We have previously looked at the contents of the $PnS attribute for
+    the test files and identified the correct label(marker):
+        - Data001.fcs: [None, None, None, None, None, None]
```

Comment:
The last element of this list should be `'Time (204.80 sec.)'`. This is what you have later in the `test_attribute()` function.